nautobot/dcim/api/views.py

```diff
@@ -146,18 +146,7 @@ class RegionViewSet(NautobotModelViewSet):
 
 
 class SiteViewSet(StatusViewSetMixin, NautobotModelViewSet):
-    queryset = (
-        Site.objects.select_related("region", "status", "tenant")
-        .prefetch_related("tags")
-        .annotate(
-            device_count=count_related(Device, "site"),
-            rack_count=count_related(Rack, "site"),
-            prefix_count=count_related(Prefix, "site"),
-            vlan_count=count_related(VLAN, "site"),
-            circuit_count=count_related(Circuit, "circuit_terminations__site"),
-            virtualmachine_count=count_related(VirtualMachine, "cluster__site"),
-        )
-    )
+    queryset = Site.objects.select_related("region", "status", "tenant").prefetch_related("tags")
```

Comment:
Just for completeness, this appears to be the correct change, removing the annotated query from this API viewset (as if it were directly copied from the UI view for `Site`), but just wanted to make sure that this was deliberate.

Reply:
Yeah it is deliberate, since `Site` model class cannot query those counts anymore